graph-manager-docs/source/integrations.md

```diff
@@ -162,24 +161,38 @@ Once you've turned on the integration in Datadog, visit the "Integrations" tab i
 
-If you want to aggregate across all operations or zoom in to a particular operation, it's simply a tag-filtering. Similarly, if you want to compare metrics across staging and production environment, it should be as simple as generating one graph per environment.
+This tagging makes it easier to see data at whatever level of granularity you desire. Whether you want to aggregate across all operations or zoom in to a particular operation, you need only choose the appropriate operation tags for filtering and the right [space aggregation function](https://docs.datadoghq.com/graphing/functions/#proceed-to-space-aggregation). Similarly, if you want to compare metrics across staging and production environments, you can filter with the appropriate variant tags.
 
-**Example**: Suppose you want to see the 95th percentile averaged across all operations for a staging and a production service.
+**Example**: Suppose you want to see the 95th percentile request latency averaged across all operations for a staging and a production service.
 
-_In the metric explorer, select `apollo.engine.operations.latency.95percentile` and then choose service where it says “one graph per” and select the two services you'd like to compare. At Apollo, we use Graph Manager to monitor Graph Manager itself, so this graph for us looks like the following_:
+_In the [Datadog metrics explorer](https://app.datadoghq.com/metric/explorer):_
+1. _Select `apollo.engine.operations.latency.95percentile` under "Graph"_
+2. _Select the targeted service name under "Over"_
+3. _Select `variant` under “One graph per” and choose the two variants for production and staging_
+4. _Select `Average of reported values` under "On each graph, aggregate with the"_
+
+_At Apollo, we use Graph Manager to monitor Graph Manager itself, so this graph for us looks like the following_:
 
 ![Compare p95](./img/datadog/datadog.png)
 
-_To perform more advanced manipulation of metrics, open up the [Metrics notebook](https://app.datadoghq.com/notebook)._
+_To perform more advanced manipulation of metrics, open up a [Datadog notebook](https://app.datadoghq.com/notebook)._
+
+#### Alerting with Datadog
+
+Graph Manager supports direct alerting on metrics via the Notifications feature, but Datadog can be a powerful partner in enabling more complex alerts via [Datadog monitors](https://docs.datadoghq.com/monitors/).
+
+**Example**: Graph Manager's Notifications feature supports alerts that trigger when the percentage of requests with error in the last 5 minutes exceeds some threshold for a specific operation. Suppose that instead of alerting on a specific operation in the last 5 minutes, we want to alert on the error percentage over all operations in some graph in the last 10 minutes, e.g. when the percentage exceeds 1% for a graph `foo` with variant `bar`.
```

Comment:
"with error" -> "with an error"

"e.g." -> "such as"

Let's name the graph `mygraph` and let's name the variant `staging` to make it feel slightly more real-world